spec/features/saml_authentication_spec.rb

```diff
@@ -65,6 +65,37 @@
     it_behaves_like "it authenticates and creates users"
   end
 
+  context "Forces a signout initiated by the IDP" do
+    before(:each) do
+      create_app('idp', %w(y))
+      create_app('sp', %w(n))
+      @idp_pid = start_app('idp', idp_port)
+      @sp_pid  = start_app('sp',  sp_port)
+    end
+    after(:each) do
+      stop_app(@idp_pid)
+      stop_app(@sp_pid)
+    end
+
+    it 'fadfasdf' do
```

Comment:
This should probably be changed :smile: 

Also, this example should work in both existing contexts, whether subject is in use or not. See above comment about using attributes.